grim.go

```diff
@@ -146,6 +151,11 @@ func (i *Instance) BuildNextInGrimQueue(logger *log.Logger) error {
 	return nil
 }
 
+func repoIsAlive(hook *hookEvent) bool {
+	url := hook.URL
+	return len(url) >= 12 && url[len(url)-12:] != "000000000000"
```

Comment:
If this current implementation ends up being the only reliable way to know if a branch has been deleted I'd recommend using this method instead of slicing the string:
http://golang.org/pkg/strings/#HasSuffix